website/pages/playground-redirect.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Redirecting…</title>
+</head>
+<body>
+  <h1>Redirecting…</h1>
+  <p>Redirecting you to the appropriate playground. If the redirect doesn’t work, <a href="javascript:history.back()">go back to the PR</a> and click the appropriate link.</p>
+  <script>
+    const match = /^https:\/\/github\.com\/prettier\/prettier\/pull\/(\d+)/.exec(document.referrer)
+    if (match != null) {
+      const [/* url */, pr] = match
+      location.replace(`https://deploy-preview-${pr}--prettier.netlify.com/playground`)
+    } else {
+      const el = document.createElement('p')
+      el.textContent = `The referrer URL (${document.referrer}) was not recognized. Sorry :(`
```

Comment:
It'd be better to also guide people to find the link here I guess:

![image](https://user-images.githubusercontent.com/8341033/47832522-ef264980-ddd0-11e8-9d84-cff44d523936.png)

Reply:
Do you have any suggestions for the wording?

Reply:
A gif should be enough:

(from https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/)

![](https://cdn.netlify.com/f0b93f3003d5669924b6e73540bbfe07bc3d8d95/34c5d/img/blog/deploy-preview-workflow.gif)

Reply:
Added. Sorry for the delay.